package proto;

message SocketClientIn {
}

message SocketClientOut {
}